test/src/main/java/org/apache/accumulo/test/replication/MultiInstanceReplicationIT.java

```diff
@@ -534,24 +534,18 @@ public void dataWasReplicatedToThePeerWithoutDrain() throws Exception {
           .setProperty(Property.REPLICATION_PEER_PASSWORD.getKey() + peerClusterName, peerPassword);
 
       String masterTable = "master", peerTable = "peer";
-
-      clientMaster.tableOperations().create(masterTable);
-      String masterTableId = clientMaster.tableOperations().tableIdMap().get(masterTable);
-      assertNotNull(masterTableId);
-
-      clientPeer.tableOperations().create(peerTable);
+      clientPeer.tableOperations().create(peerTable, new NewTableConfiguration());
       String peerTableId = clientPeer.tableOperations().tableIdMap().get(peerTable);
       assertNotNull(peerTableId);
 
-      // Give our replication user the ability to write to the table
-      clientPeer.securityOperations().grantTablePermission(peerUserName, peerTable,
```

Comment:
Is this present in the new changes?  If not, was it intentionally deleted?  I got kinda lost in the diff for this file.

Reply:
good catch.  I got lost in a few of these too.  The test seemed to run anyway.  I will double check that.

Reply:
I am not sure if its needed, I did not look that closely at the test.  I was just looking at the diff for correctness of table names and props.